pgkit/application/models/replica.py

```diff
@@ -257,7 +257,8 @@ def _get_current_delay(self):
         config = read_file(file_location)
         for line in config.split('\n'):
             if 'recovery_min_apply_delay' in line:
-                return int(line.split(' ')[2].split("min")[0]) / 60
+                time_difference = line.split(' ')[2].split("min")[0]
+                return int(0 if time_difference=="''" else int(time_difference)) / 60
```

Comment:
Please check whether a `recovery_min_apply_delay` equal to '' is valid for PostgreSQL; if not we can assume that time_difference is always a number for valid PostgreSQL configurations unless the line is commented.

Reply:
According to postgres docs, recovery_min_apply_delay must be an integer and '' is not valid. I think your assumption is true and instead of the code above, we can check that the line is commented or not.

Reply:
We now use line.startswith in the if condition. This resolves the empty string case.